### DISCUSSION TITLE
Make library smaller

### DISCUSSION THREAD

**Discussion by damjanmi** [Taiizor/ReaLTaiizor#113] [Q&A] [answered]
Hi,

Is it possible to make library smaller? Let's say that I only use Material form.

Damjan

**Comment by Taiizor** [comment, marked as answer]
Its possible, but very tricky. I thought of presenting each theme in different libraries, but I did not go under this burden.
Of course, you can fork the project and just reserve the Material theme for yourself. But i wouldnt recommend it.
